src/components/CollectiveCard.js

```diff
@@ -96,7 +96,8 @@ class CollectiveCard extends React.Component {
                 vertical-align: top;
                 position: relative;
                 box-sizing: border-box;
-                width: 200px;
```

Comment:
This will impact many places of the website. However after testing I don't see any blockers to not do it, it looks good and will improve consistency.